lib/types/local-date.js

```diff
@@ -1,32 +1,42 @@
 "use strict";
 const utils = require("../utils");
+const rust = require("../../index");
 /** @module types */
 
 /**
+ * 2^31 days before unix epoch is -5877641-06-23. This is the first day that can be represented by this class.
  * @private
  * @const
  */
-const millisecondsPerDay = 86400000;
+const dateCenter = Math.pow(2, 31);
+
 /**
  * @private
+ * @const
  */
-const dateCenter = Math.pow(2, 31);
+const millisecondsPerDay = 86400000;
```

Comment:
If it is private use #varName

Reply:
My mistake. A variable that is not in a class cannot be private....